patches/chrome-browser-resources-tools-rollup_plugin.js.patch

```diff
@@ -0,0 +1,62 @@
+diff --git a/chrome/browser/resources/tools/rollup_plugin.js b/chrome/browser/resources/tools/rollup_plugin.js
+index e1942718e76890c5051923fc647f488c1801ca95..48d88bbd38e5f626a0466ae73413ae5595bffc59 100644
+--- a/chrome/browser/resources/tools/rollup_plugin.js
++++ b/chrome/browser/resources/tools/rollup_plugin.js
+@@ -39,6 +39,47 @@ function combinePaths(origin, source) {
+   return normalizeSlashes(path.normalize(path.join(originDir, source)));
+ }
+ 
++function braveResolveId(params, source, origin) {
```

Comment:
@bridiver @petemill I've cut the size/impact of the patch down significantly. Will continue to look into a way to make this better. I might be able to drop this function into a `chromium_src` override and then edit `optimize_webui.py` to add pathing (so it can be included via `require`)